packages/nomad/main.tf

```diff
@@ -186,28 +186,31 @@ data "google_compute_machine_types" "client" {
   filter = "name = \"${var.client_machine_type}\""
 }
 
-resource "nomad_job" "orchestrator" {
-  jobspec = file("${path.module}/orchestrator.hcl")
-
-  hcl2 {
-    vars = {
-      gcp_zone     = var.gcp_zone
-      port         = var.orchestrator_port
-      environment  = var.environment
-      consul_token = var.consul_acl_token_secret
-      cpu_mhz      = var.environment == "prod" ? floor(data.google_compute_machine_types.client.machine_types[0].guest_cpus * 1.5) * 1000 : 1000
-      memory_mb    = var.environment == "prod" ? floor(data.google_compute_machine_types.client.machine_types[0].memory_mb * 0.6 / 1024) * 1024 : 1024
```

Comment:
I removed these 2 because, they are not doing anything, just causing troubles, if you would like to change machine size and do something like opportunistic deploy, you can't